src/fsharp/vs/Symbols.fs

```diff
@@ -523,10 +523,22 @@ and FSharpEntity(cenv:cenv, entity:EntityRef) =
     member x.RecordFields = x.FSharpFields
     member x.FSharpFields =
         if isUnresolved() then makeReadOnlyCollection[] else
+    
+        if entity.IsILEnumTycon then
+            let (TILObjectReprData(scoref,enc,tdef)) = entity.ILTyconInfo
```

Comment:
I think these next few lines can be replaced by a use of ``generalizeTyconRef``?